app/presenters/publishing_api/call_for_evidence_presenter.rb

```diff
@@ -0,0 +1,243 @@
+module PublishingApi
+  class CallForEvidencePresenter
+    include UpdateTypeHelper
+
+    SCHEMA_NAME = "call_for_evidence".freeze
+
+    attr_reader :update_type
+
+    delegate :content_id, to: :call_for_evidence
+
+    def initialize(call_for_evidence, update_type: nil)
+      self.call_for_evidence = call_for_evidence
+      self.update_type = update_type || default_update_type(call_for_evidence)
+    end
+
+    def content
+      BaseItemPresenter
+        .new(call_for_evidence, update_type:)
+        .base_attributes
+        .merge(PayloadBuilder::AccessLimitation.for(call_for_evidence))
+        .merge(PayloadBuilder::PublicDocumentPath.for(call_for_evidence))
+        .merge(
+          description: call_for_evidence.summary,
+          details:,
+          document_type:,
+          public_updated_at:,
+          rendering_app: Whitehall::RenderingApp::GOVERNMENT_FRONTEND,
+          schema_name: SCHEMA_NAME,
+          links: edition_links,
+          auth_bypass_ids: [call_for_evidence.auth_bypass_id],
+        )
+    end
+
+    def links
+      # TODO: Previously, this presenter was sending all links to the
+      # Publishing API at both the document level, and edition
+      # level. This is probably redundant, and hopefully can be
+      # improved.
+      edition_links
+    end
+
+    def edition_links
+      LinksPresenter
+        .new(call_for_evidence)
+        .extract(%i[organisations parent topics government])
+        .merge(PayloadBuilder::People.for(call_for_evidence))
+        .merge(PayloadBuilder::Roles.for(call_for_evidence))
+        .merge(PayloadBuilder::TopicalEvents.for(call_for_evidence))
+    end
+
+  private
+
+    attr_accessor :call_for_evidence
+    attr_writer :update_type
+
+    delegate :display_type_key, to: :call_for_evidence
+    alias_method :document_type, :display_type_key
+
+    def base_details
+      {
+        body:,
+        closing_date: call_for_evidence.closing_at,
+        emphasised_organisations: call_for_evidence.lead_organisations.map(&:content_id),
+        opening_date: call_for_evidence.opening_at,
+      }
+    end
+
+    def body
+      Whitehall::GovspeakRenderer.new.govspeak_edition_to_html(call_for_evidence)
+    end
+
+    def details
+      base_details
+        .merge(ChangeHistory.for(call_for_evidence))
+        .merge(Documents.for(call_for_evidence))
+        .merge(ExternalURL.for(call_for_evidence))
+        .merge(NationalApplicability.for(call_for_evidence))
+        .merge(WaysToRespond.for(call_for_evidence))
+        .merge(PayloadBuilder::FirstPublicAt.for(call_for_evidence))
+        .merge(PayloadBuilder::PoliticalDetails.for(call_for_evidence))
+        .merge(PayloadBuilder::TagDetails.for(call_for_evidence))
+        .merge(PayloadBuilder::Attachments.for(call_for_evidence))
+    end
```

Comment:
Do we need to add an outcome to this list to mirror the consultation's final outcome? 

https://github.com/alphagov/whitehall/blob/main/app/presenters/publishing_api/consultation_presenter.rb#L77